pegasus/sites.v3/code.org/public/afe/benefits.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Amazon Future Engineer Ineligible
+theme: responsive
+---
+
+You're ineligible :(
```

Comment:
I'm guessing this will be updated later :)

Reply:
Haha yep!